nlpretext/_utils/daskloader.py

```diff
@@ -0,0 +1,19 @@
+# mypy: disable-error-code="attr-defined"
+import dask.bag as db
+import dask.dataframe as dd
+
+
+def read_text(files_path, encoding):
+    return db.read_text(files_path, encoding=encoding).str.strip().to_dataframe()
+
+
+def read_json(files_path, encoding):
+    return dd.read_json(files_path, encoding=encoding)
+
+
+def read_csv(files_path, encoding):
+    return dd.read_csv(files_path, encoding=encoding)
+
+
+def read_parquet(files_path, encoding):
+    return dd.read_parquet(files_path, encoding=encoding)
```

Comment:
Even though it's already clear, type hinting and a small docstring would be nice I think

Reply:
Ok for type hints, but i tried to add docstrings and it only makes the code more cumbersome